activerecord/lib/active_record/railtie.rb

```diff
@@ -102,6 +102,17 @@ class Railtie < Rails::Railtie # :nodoc:
       end
     end
 
+    initializer "active_record.warn_on_result_set_size" do
+      if config.active_record.warn_on_result_set_size
+        config.after_initialize do |app|
```

Comment:
It's not needed, its only there because I copied and pasted from the previous `initializer`, doh! Will remove